businessRules/users.js

```diff
@@ -2,14 +2,21 @@ const { v4 } = require('uuid');
 const md5 = require('md5');
 const { createUser } = require('../models/users');
 const { getUsers, setUsers } = require('../utils/storage');
+const {
+    validateUserName,
+    validatePassword,
+    findUserWithSameUserNameAndPassword,
```

Comment:
The `findUserWithSameUserNameAndPassword` name is too verbose. Can you improve it?

Reply:
Tryed to rename.